get-docker.sh

```diff
@@ -0,0 +1,741 @@
+#!/bin/sh
```

Comment:
which is the role of this script here?

Reply:
I actually don't know. Is it possible that this gets generated when running the initialize.sh script?

Reply:
Hi :)
yes, you're right. It is downloaded in `initialize.sh` script and never removed.
Can you please remove script from your PR and add an `rm` to `initialize.sh` script ?

tysm for pointing out :)

Reply:
I've added the removal of the script. 

I've added it with sudo as it also gets run with sudo - hope that is ok.

Reply:
Thank you! I think you can delete it without `sudo` as it gets downloaded with the same permission of the user launching `curl`.

Reply:
Okai, I'll fix that and maybe the rest I mentioned above. 👍🏻